packages/strapi-plugin-users-permissions/services/Providers.js

```diff
@@ -392,3 +393,11 @@ const getProfile = async (provider, query, callback) => {
       break;
   }
 };
+
+const buildRedirectUri = (provider = '') =>
+  `${getAbsoluteServerUrl(strapi.config)}/connect/${provider}/callback`;
```

Comment:
Could somebody explain why this remains pointing to _/connect/..._ . Shouldn't that point to _/auth/..._ as this is the route configured [here](https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-users-permissions/config/routes.json#L235)

Reply:
We use the lib `grant` and it generatse itself the redirect_uri (as it is it itself that needs the answer from the provider).
In the admin panel, we generate the `redirect_uri` the same way grant does.
We can see it in its doc here: https://create-react-app.dev/docs/adding-custom-environment-variables/ at the line `redirect_uri`.